gui/webdisplay/src/TWebWindowsManager.cxx

```diff
@@ -449,26 +475,25 @@ bool ROOT::Experimental::TWebWindowsManager::Show(ROOT::Experimental::TWebWindow
 
       if (symbol_cef) {
 
-         if (win.IsBatchMode()) {
+         if (batch_mode) {
```

Comment:
The `#ifdef R__HAS_CEFWEB` and similar parts look like good candidates for splitting into separate functions?

Reply:
But exactly because of ifdef I do not like to split. 
That to sense declare function for the module (CEF), which never appears for normal user?

Reply:
Private member function, or even `static`, source-only function will do. I'm not saying that this should become part of the class's interface. But I understand your argument that this function will still evolve.